app/modules/rfswitch.c

```diff
@@ -76,19 +98,25 @@ void transmit(HighLow pulses, bool invertedSignal, int pulseLength, int pin) {
  */
 void send(unsigned long protocol_id, unsigned long pulse_length, unsigned long repeat, unsigned long pin, unsigned long value, unsigned int length) {
   platform_gpio_mode(pin, PLATFORM_GPIO_OUTPUT, PLATFORM_GPIO_FLOAT);
-  Protocol p = proto[protocol_id-1];
+  Protocol p = custom;
+  if(protocol_id > 0) proto[protocol_id-1];
   for (int nRepeat = 0; nRepeat < repeat; nRepeat++) {
+
+    if((p.frameStartStop & 1) > 0) transmit(p.invertStartStop ? p.one : p.zero, p.invertedSignal, pulse_length, pin);
+
     for (int i = length-1; i >= 0; i--) {
       if (value & (1L << i))
         transmit(p.one, p.invertedSignal, pulse_length, pin);
       else
         transmit(p.zero, p.invertedSignal, pulse_length, pin);
     }
+    if((p.frameStartStop & 2) > 0) transmit(p.invertStartStop ? p.one : p.zero, p.invertedSignal, pulse_length, pin);
+
     transmit(p.syncFactor, p.invertedSignal, pulse_length, pin);
   }
+  platform_gpio_write(pin, 0);      // If we leave gpio 0 high on last transmit, we flood the rf band (so we should always set low this after sending waveforms)
```

Comment:
Looking back at the previous fix #1738, the correct thing to do here might be to set the GPIO pin state to p.invertedSignal

`platform_gpio_write(pin, p.invertedSignal); `

Reply:
I reviewed this again, and in one of my usage scenarios I actually set invertedSignal to true. Since my hardware's 433 transmitter is active high, it should still 0 as I have set it here. @ffedoroff who authored #1738 might be able to comment on whether or not this is still sufficient for his scenario given he originally moved away from this approach.